tests/vmi_multus_test.go

```diff
@@ -658,6 +668,19 @@ var _ = Describe("SRIOV", func() {
 			return
 		}
 
+		createSRIOV_VMI := func(networksNames []string) (vmi *v1.VirtualMachineInstance) {
+			SRIOV_VMI := getSriovVmi(networksNames)
+
+			// This step for the three vms is needed to guarantee that no VFs on the PF carry a duplicate MAC address that may affect
```

Comment:
please omit the part about the '3 vms'. This is located within a function; doesn't make sense to specify the number of times you'll be using it now. Just omitting 'for the three vms' would do it, since it reads well.